src/xhydro/modelling/_hydrotel.py

```diff
@@ -409,11 +410,25 @@ def _standardise_outputs(self, **kwargs):
                 stdout = subprocess.check_output(  # noqa: S603
                     [self.executable], stdin=subprocess.DEVNULL, text=True
                 )
-            ds.attrs["Hydrotel_version"] = str(stdout).split("HYDROTEL ")[1].split("\\n")[0]
+            hydrotel_version = re.search(r"HYDROTEL \d\.\d\.\d.\d{4}", stdout)
+            if hydrotel_version is not None:
+                ds.attrs["Hydrotel_version"] = hydrotel_version.group(0).split(" ")[1]
+            else:
+                ds.attrs["Hydrotel_version"] = "unspecified"
             ds.attrs["Hydrotel_config_version"] = self.simulation_config["SIMULATION HYDROTEL VERSION"]
 
             # Overwrite the file
-            chunks = estimate_chunks(ds, dims=["subbasin_id"], target_mb=5)
+            # If the file is larger than 100 MB, rechunk it to ~25 MB chunks along the 'subbasin_id' dimension
+            ds_size_mb = ds["q"].size * 4 / 1024 / 1024
+            if ds_size_mb > 100:
+                chunks = estimate_chunks(ds, dims=["subbasin_id"], target_mb=25)
+                # FIXME: This is fixed in the latest version of xscen. Remove this workaround once we depend on it.
```

Comment:
You could write the new logic in here with a conditional for the new `xscen` version; We're due for a release anyway.